From d0356e59b4e51fb87f477faa78fd28d6e35d84f1 Mon Sep 17 00:00:00 2001
From: allo-com <jaikumar@cem-solutions.net>
Date: Mon, 29 Jul 2019 15:06:57 +0530
Subject: [PATCH 722/782] codecs: Correct Katana minimum volume

Update Katana minimum volume to get the exact 0.5 dB value in each step.

Signed-off-by: Sudeep Kumar <sudeepkumar@cem-solutions.net>
---
 sound/soc/bcm/allo-katana-codec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/sound/soc/bcm/allo-katana-codec.c
+++ b/sound/soc/bcm/allo-katana-codec.c
@@ -126,7 +126,7 @@ static SOC_VALUE_ENUM_SINGLE_DECL(katana
 				  katana_codec_deemphasis_texts,
 				  katana_codec_deemphasis_values);
 
-static const SNDRV_CTL_TLVD_DECLARE_DB_MINMAX(master_tlv, -12700, 0);
+static const SNDRV_CTL_TLVD_DECLARE_DB_MINMAX(master_tlv, -12750, 0);
 
 static const struct snd_kcontrol_new katana_codec_controls[] = {
 	SOC_DOUBLE_R_TLV("Master Playback Volume", KATANA_CODEC_VOLUME_1,
